### ISSUE TITLE
Compatibility with CKEditor 5 custom field ?

### ISSUE THREAD

**Issue by copostic** [Fekide/strapi-plugin-translate#41]
Hi guys,

I was using [Strapi Plugin CKEditor](https://github.com/nshenderov/strapi-plugin-ckeditor) until I found out it was not maintained that much and above all that an [official CKEditor plugin](https://market.strapi.io/plugins/@ckeditor-strapi-plugin-ckeditor) was available.

I installed it and replaced my richtext fields by customField eg:
<br>
**Before:** 
```JSON
{
    "excerpt": {
      "pluginOptions": {
        "i18n": {
          "localized": true
        }
      },
      "type": "richtext",
      "required": true
    },
}
```
**After:**
```JSON
{    
"excerpt": {
      "pluginOptions": {
        "i18n": {
          "localized": true
        },
        "deepl": {
          "translate": "translate"
        }
      },
      "type": "customField",
      "options": {
        "preset": "rich"
      },
      "customField": "plugin::ckeditor.CKEditor",
      "required": true
    }

}
```   

But when I try to translate from another locale, I have this error : 
![image](https://user-images.githubusercontent.com/17200174/210539943-2cfacd80-25e6-4a98-adea-3bdb1d71901f.png)

Any idea why ?

Thanks !

**Comment by copostic** [comment]
Ah it seems to be related to asset mime so probably nothing to do with CKEditor but more #39

**Comment by feliche93** [comment]
@sargreal I also installed the plugin mentioned above and get the following error:

```
error: strapi.service(...).create is not a function
TypeError: strapi.service(...).create is not a function
    at BatchTranslateJob.start (/Users/felixvemmer/Developer/cryptoticker/node_modules/strapi-plugin-deepl/server/services/batch-translate/BatchTranslateJob.js:216:48)
This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
Error: Translation of an entity failed
    at BatchTranslateJob.start (/Users/felixvemmer/Developer/cryptoticker/node_modules/strapi-plugin-deepl/server/services/batch-translate/BatchTranslateJob.js:236:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

The exception occurrs in this line:

```
        // Create localized entry
        await strapi.service(this.contentType).create({
          data: fullyTranslatedData,
          // Needed for syncing localizations
          populate: ['localizations'],
        })
```

Would love to provide more info to debug this further.

**Comment by layaxx** [comment]
@feliche93 I think it would be best if you could open a new Issue with the Bug report template and provide the information requested there, then @sargreal and I can take a look when one of us got time ^^

**Comment by feliche93** [comment]
@layaxx, done ✅ thanks for any help or debugging assistance: https://github.com/Fekide/strapi-plugin-translate/issues/121#issue-1604685566

**Comment by sargreal** [comment]
Closing due to most likely have been solved in #42
